site/ja/docs/privacy-sandbox/first-party-sets/index.md

```diff
@@ -67,19 +69,17 @@ First-Party Sets を補完する提案は、`SameParty` Cookie 属性です。 C
 
 つまり、b.example または c.example の訪問者が a.example に対してリクエストを行うと、`セッション` Cookie がそのリクエストに含まれます。
 
-
 ---
 
 ## 貢献とフィードバックの共有
 
-* **オリジントライアル**: First-Party Sets と SameParty の最初の[オリジントライアル](/origintrials/#/view_trial/988540118207823873)は Chrome バージョン 89 ～ 93 で公開されていましたが、現在は終了しています。
-* **GitHub**: [提案](https://github.com/privacycg/first-party-sets)を読んで、[質問を挙げ、ディスカッションをフォロー](https://github.com/privacycg/first-party-sets/issues)できます。
-* **開発者サポート**: [Privacy Sandbox Developer Support リポジトリ](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support)では、質問したり、ディスカッションに参加したりできます。
-
+- **オリジントライアル**: First-Party Sets と SameParty の最初の[オリジントライアル](/origintrials/#/view_trial/988540118207823873)は Chrome バージョン 89 ～ 93 で公開されていましたが、現在は終了しています。
+- **GitHub**: [提案](https://github.com/privacycg/first-party-sets)を読んで、[質問を挙げ、ディスカッションをフォロー](https://github.com/privacycg/first-party-sets/issues)できます。
+- **開発者サポート**: [Privacy Sandbox Developer Support リポジトリ](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support)では、質問したり、ディスカッションに参加したりできます。
 
 ## 詳細
 
-* [First-Party Sets 属性と sameParty 属性](/blog/first-party-sets-sameparty/)
-* [First-Party Sets の技術 Explainer](https://github.com/privacycg/first-party-sets)
-* [Chrome プラットフォームのステータス](https://chromestatus.com/feature/5640066519007232)
-* [Chromium プロジェクト](https://www.chromium.org/updates/first-party-sets)
+- [First-Party Sets 属性と sameParty 属性](/blog/first-party-sets-sameparty/)
+- [First-Party Sets の技術 Explainer](https://github.com/privacycg/first-party-sets)
+- [Chrome プラットフォームのステータス](https://chromestatus.com/feature/5640066519007232)
+- [Chromium プロジェクト](https://www.chromium.org/updates/first-party-sets)
```

Comment:
I didn't change any of lines above which turns `*` into `-`. LIne 76-78 and 82-85. Are these done by GitLocalize?

Reply:
Correct, by GitLocalize. Both `*` and `-` are the correct notations for markdown bullet lists, and GL's reverse parser uses the `-` notation. 
We figured this shouldn't be a problem as long as 11ty renders correct html from this (which it does): 
<img width="580" alt="Screenshot 2022-08-26 at 15 59 11" src="https://user-images.githubusercontent.com/8548803/186909078-9a4cfb9b-a5cf-4018-8bda-13b87aa4d09d.png">

If there's another reason this is important, please let me know and we'll look into this with a dev team (or set up a GitHub action on our fork to correct this automatically).

Reply:
I agree this shouldn't be a problem, but I'm curious why this conversion occurred if GitLocalize only translates updated fragments. Also, I didn't touch the section at all. When did this conversion occur?

Reply:
I see. Let's revisit when this causes any problems.